src/applications/vaos/referral-appointments/ChooseDateAndTime.unit.spec.jsx

```diff
@@ -95,14 +95,17 @@ describe('VAOS ChooseDateAndTime component', () => {
     featureToggles: {
       vaOnlineSchedulingCCDirectScheduling: true,
     },
-    appointmentApi: {
-      mutations: {
-        postDraftReferralAppointmentCache: {
-          status: 'fulfilled',
-          data: createDraftAppointmentInfo(1),
-        },
-      },
+    referral: {
+      draftAppointmentInfo: createDraftAppointmentInfo(1),
     },
+    // appointmentApi: {
```

Comment:
Can we add a test that tests the mutation too? Could it catch stuff like that missing parameter? Maybe that's something you could catch in the cypress tests which also passed 🤔

Reply:
For both Review and Choose, I updated the called test to be calledWith and passed it the specific parameters we are expecting to make sure it fails correctly. I tested with my code before reverting the empty string to make sure it would fail at that point.